test/integration/students/student_sees_events_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+
+class StudentSeesEventsTest < ActionDispatch::IntegrationTest
+
+  def create_events
+    3.times do |x|
+      date = Time.now
+      Event.create(name: "Event#{x}",
+                   description: "Here is a description for event#{x}",
+                   location: Faker::Address.street_address,
+                   starts_at: date,
+                   ends_at: date) 
+    end
+  end
+
+  def login_student
+    FactoryGirl.create(:user, :default)
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.